spec/features/articles_list_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'ffaker'
```

Comment:
this line should actually be in `spec/factories/articles.rb` (and other respective factories as needed)